manifests/f/Filius/Filius/2.1.0/Filius.Filius.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Filius.Filius
+PackageVersion: 2.1.0
+Installers:
+- Architecture: neutral

```

```suggestion
- Architecture: x64
```
We've moved away from using `neutral` as architecture. If this application runs fine on x86 or other arm architectures, you can add another Installer node.